apps/docs/components/Clippy/ClippyModal.tsx

```diff
@@ -32,13 +41,20 @@ const ClippyModal: FC<Props> = ({ onClose }) => {
     setAnswer(response.data.answer)
   }, [query, supabaseClient])
 
+  function handleQueryPreload(question: string) {
+    setQuery(question)
+    handleConfirm()
+  }
+
+  function handleResetPrompt() {
+    setQuery('')
+    setAnswer(undefined)
+  }
+
   return (
-    <div
-      className="flex flex-col items-center fixed top-0 left-0 w-screen h-screen p-4 md:p-[12vh] backdrop-blur-sm z-50 cursor-pointer bg-black/40 dark:bg-black/30 "
-      onClick={onClose}
-    >
+    <Modal size="xlarge" visible={true} onCancel={onClose} hideFooter closable={false}>
       <div
-        className={`prose dark:prose-dark mx-auto flex flex-col gap-4 rounded-lg p-6 w-full max-w-3xl shadow-2xl overflow-hidden border text-left border-scale-500 bg-scale-300 cursor-auto`}
```

Comment:
@saltcod we seemed to need this to get the markdown formatting to work across various components (inline code snippets, links, etc). I noticed `DefaultLayout.tsx` has this, which isn't applying here because this modal is placed in a React portal outside the tree. Maybe there's a better way.

Reply:
Yep looks like.